nda-transform/count_assoc_files.py

```diff
@@ -1,81 +1,138 @@
+            df = pd.read_csv(file, header=1)
+        else:
+            nda_submissions_path = project_path / "nda-submissions"
+            df = pd.read_csv(file)
+            _data_file_path = nda_submissions_path / df.loc[0]["SUBMISSION_FOLDER_NAME"]
+
+        # check a file only if at least one file_columns exist
+        check_file = False
         for c in file_columns:
-        
             if c in df.columns:
-                if not pd.isna(row[c]):
-                    data_file = row[c]
-
-                    data_file_path: Path = _data_file_path / data_file
-
-                    if not data_file_path.exists():
-                        print(f"File does not exist: {data_file_path}")
-                    else:
-                        count+=1
-
-                        if data_file_path not in unique_file:
-                            unique_file[data_file_path]=1
+                check_file = True
+                break
+
+        if not check_file:
+            print(": Expected columns not found in the file. Skipping.")
+            continue
+        else:
+            print()
+
+        for _, row in df.iterrows():
+            for c in file_columns:
+                if c in df.columns:
+                    if not pd.isna(row[c]):
+                        data_file = row[c]
+                        data_file_path: Path = _data_file_path / data_file
+
+                        if not data_file_path.exists():
+                            print(f"File does not exist: {data_file_path}")
                         else:
-                            # count duplicates
-                            manifest=file.name
-                            if manifest in duplicate:
-                                duplicate[manifest]+=1
+                            files_count += 1
+                            if data_file_path not in unique_files:
+                                unique_files.add(data_file_path)
                             else:
-                                duplicate[manifest]=1
-
+                                # count duplicates
+                                manifest = file.name
+                                if manifest in duplicate_files_count:
+                                    duplicate_files_count[manifest] += 1
+                                else:
+                                    duplicate_files_count[manifest] = 1
 
+    print(f"Total files: {files_count}")
+    print(f"Total unique files: {len(unique_files)}")
+    print(f"Duplicates: {duplicate_files_count}")
 
-print('Total files', count)
-print('Total unique files', len(unique_file))
-print('Duplicates', duplicate)
 
+# Usage:
+# at AWS workspace:
+# __file__  -id 65021 -collection 3705
+# at ERIS cluster:
+# __file__ -id 0 -collection 0
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Count unique and duplicate files")
+    parser.add_argument("-id", type=str, help="submission id", default="0")
```

Comment:
Change to `('-i', '--id',` . Remember, the convention is that you use `--` for long argument and `-` for single letter argument.